src/Components/Todo.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Todo({ lists, completeTodo }) {
```

Comment:
Like its weird to think of this component as reusable... It contains all the list items *inside* it